glue-codes/labview/CMakeLists.txt

```diff
@@ -14,15 +14,26 @@
 # limitations under the License.
 #
 
+add_library(dummylib SHARED
```

Comment:
Is this needed anymore?  I don't see it used anywhere.

Reply:
Will remove later: https://github.com/orgs/OpenFAST/projects/9/views/1?pane=issue&itemId=108648606